packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5856,11 +5856,22 @@ export default class Meeting extends StatelessWebexPlugin {
           this
         );
 
-        joinFailed(error);
+        const proxyError = new Proxy(error, {
+          // eslint-disable-next-line require-jsdoc
+          get(target, prop) {
+            if (prop === 'handledBySdk') {
```

Comment:
I wonder if we should add a few more (other iterations) to pull the `category` from it?

Reply:
i.e., 

`handledBySdk`,
`errorCategoryCA`,
`telemetrySent` 

? idk something to brew on but not needed